survey/HTML-syntax-in-EPUB.md

```diff
@@ -0,0 +1,20 @@
+---
+title: Survey on the Impact of Allowing HTML Syntax in EPUB
+date: 2025-05-08
+---
+
+The Publishing Maintenance Working Group (PMWG) is exploring the possibility of allowing the [HTML syntax](https://html.spec.whatwg.org/multipage/syntax.html#syntax) for the HTML content document format in EPUB 3.4, instead of restricting it to the [XML Syntax](https://html.spec.whatwg.org/multipage/xhtml.html#the-xhtml-syntax) as it is in earlier versions of EPUB 3.
```

Comment:
```suggestion
The Publishing Maintenance Working Group (PMWG) plans to allow the [HTML syntax](https://html.spec.whatwg.org/multipage/syntax.html#syntax) for the HTML content document format in EPUB 3.4, instead of restricting it to the [XML Syntax](https://html.spec.whatwg.org/multipage/xhtml.html#the-xhtml-syntax) as it was done for earlier versions of EPUB 3.
```

I think what we said on the call is that we do plan to do this, but we have to find the best way of doing it. Hence, the proposed change.